be-dataapp_resources/application-docker.properties

```diff
@@ -1,11 +1,12 @@
 ### App setup
 spring.banner.location=classpath:banner.txt
 server.port=8083
-server.ssl.key-alias=ssl
+server.ssl.key-alias=execution-core-container
 server.ssl.key-password=changeit
 server.ssl.key-store=classpath:ssl-server.jks
```

Comment:
Just to be aligned, can we put this cert to be the same like in ECC, in order to be sure that hostname validation will be correct? For example:
/ecc_cert/ssl-server.jks 
Maybe this will require to also remove jks from dataApp also.
Probably we will have to mount ecc_cert to dataApp image, so we have access to jks